src/react/PropTypes.js

```diff
@@ -1,14 +1,12 @@
-import {PropTypes} from "react";
+import PropTypes from "prop-types";
 
```

Comment:
If you want me to switch back to `const {shape, object, string} = PropTypes;`  I can.  I was just following a style that I use, dot notation from `PropTypes` namespace.  I think you use it here too. https://github.com/nfl/react-metrics/blob/master/test/ReactMetrics/MetricsElement.spec.js#L118